setup.py

```diff
@@ -52,6 +52,7 @@
         'ruamel.yaml',
         'scikit-image',
         'scikit-learn',
+        'SkillMetrics',
```

Comment:
```suggestion
```

I don't find a reference to this package in any of the new diagnostics, so this can probably be removed.